src/core/qgssnappingconfig.h

```diff
@@ -95,7 +95,7 @@ class CORE_EXPORT QgsSnappingConfig
     /**
      * Convenient method to return an icon corresponding to the enum type
      * Qgis::SnappingTypes.
-     * \deprecated QGIS 3.26. Use Qgis::snappingTypeToString instead.
+     * \deprecated QGIS 3.26. Use Qgis::snappingTypeToIcon instead.
```

Comment:
```suggestion
     * \deprecated QGIS 3.26. Use snappingTypeToIcon() instead.
```

@nyalldawson It looks like there is some not wanted `Qgis::` inserted  in this file no?

Reply:
Not sure what's occurring there -- but all those Qgis:: should be removed. I suspect a bad copy/replace at some stage when something was moved.

Reply:
i don't know about the standards, but the Qgis. indications seem useful as the deprecated method is in QgsSnappingConfig not Qgis ?

Reply:
default is to look in the same class, so snappingTypeToIcon() refers to QgsSnappingConfig::snappingTypeToIcon().

As @nyalldawson pointed out, we should get rid of Qgis::

Reply:
The methods are in the same class not Qgis right, i got myself confused. I fixed this for the 2 similar functions.